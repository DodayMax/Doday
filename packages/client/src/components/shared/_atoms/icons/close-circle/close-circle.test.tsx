import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('CloseCircle', () => {
  it('renders correctly without props', () => {
    const icon = shallow(<Icons.CloseCircle />);

    expect(icon).toMatchSnapshot();
  });
});
