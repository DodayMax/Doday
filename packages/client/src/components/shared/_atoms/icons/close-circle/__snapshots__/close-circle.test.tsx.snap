// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseCircle renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <g
        fill="none"
        fillRule="evenodd"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <g
          fill="black"
        >
          <path
            d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
          />
          <path
            d="M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z"
          />
        </g>
      </g>,
      "height": 20,
      "viewBox": "0 0 24 24",
      "width": 20,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />,
          <g
            fill="black"
          >
            <path
              d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
            />
            <path
              d="M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z"
            />
          </g>,
        ],
        "fill": "none",
        "fillRule": "evenodd",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M0 0h24v24H0z",
            "fill": "none",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <path
                d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
              />,
              <path
                d="M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z"
              />,
            ],
            "fill": "black",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
          ],
          "type": "g",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <g
            fill="black"
          >
            <path
              d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
            />
            <path
              d="M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z"
            />
          </g>
        </g>,
        "height": 20,
        "viewBox": "0 0 24 24",
        "width": 20,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />,
            <g
              fill="black"
            >
              <path
                d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
              />
              <path
                d="M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z"
              />
            </g>,
          ],
          "fill": "none",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M0 0h24v24H0z",
              "fill": "none",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <path
                  d="M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z"
                />,
                <path
                  d="M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z"
                />,
              ],
              "fill": "black",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "d": "M12.108 1.196c-5.94 0-10.756 4.815-10.756 10.755s4.815 10.756 10.756 10.756c5.94 0 10.755-4.816 10.755-10.756S18.048 1.196 12.108 1.196zm0 20.044a9.289 9.289 0 1 1 0-18.578 9.289 9.289 0 0 1 0 18.578z",
                },
                "ref": null,
                "rendered": null,
                "type": "path",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "d": "M16.038 8a.733.733 0 0 0-1.036 0l-2.782 2.895-3.026-2.933a.733.733 0 1 0-1.017 1.056l3.007 2.894-3.007 2.894a.733.733 0 1 0 1.017 1.056l3.026-2.933 2.782 2.894a.733.733 0 1 0 1.056-1.017l-2.802-2.933 2.802-2.933a.733.733 0 0 0-.02-.94z",
                },
                "ref": null,
                "rendered": null,
                "type": "path",
              },
            ],
            "type": "g",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
